src/main/java/life/qbic/io/commandline/UploadPetabResultCommand.java

```diff
@@ -0,0 +1,79 @@
+package life.qbic.io.commandline;
+
+import ch.ethz.sis.openbis.generic.OpenBIS;
+import ch.ethz.sis.openbis.generic.asapi.v3.dto.dataset.id.DataSetPermId;
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import life.qbic.App;
+import life.qbic.io.PetabParser;
+import life.qbic.model.download.OpenbisConnector;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command(name = "upload-petab-result",
+    description = "uploads a petab based on other PETab downloaded from openbis and attaches it to a provided experiment and any datasets referenced in the PETab metadata.")
+public class UploadPetabResultCommand implements Runnable {
+
+  @Parameters(arity = "1", paramLabel = "file/folder", description = "The path to the file or folder to upload")
+  private String dataPath;
+  @Parameters(arity = "1", paramLabel = "experiment ID", description = "The full identifier of the experiment the data should be attached to. "
+      + "The identifier must be of the format: /space/project/experiment")
+  private String experimentID;
+  //@Option(arity = "1..*", paramLabel = "<parent_datasets>", description = "Optional list of dataset codes to act"
+  //    + " as parents for the upload. E.g. when this dataset has been generated using these datasets as input.", names = {"-pa", "--parents"})
```

Comment:
Why was this commented out? Can this be removed?

Reply:
the ids are now provided via the yaml file, but there might be feedback that this option is better